pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -165,7 +165,6 @@ def replay_f2c(args: list[str], dryrun: bool = False) -> list[str] | None:
         found_source = True
 
     if not found_source:
-        print(f"f2c: source not found, skipping: {new_args_str}")
```

Comment:
Does this happen? Is there a reason we don't want it to log this?

Reply:
During the setup step, meson calls `gfortran` (and other compilers as well) multiple times to test the compiler, like checking versions, compiling sample codes, etc. So yes, this can happen.

I removed the log because it was preventing meson from parsing the value from the result of gfortran's execution.